routes/api.php

```diff
@@ -196,6 +196,7 @@
     Route::post('tasks/{task}/rollback', [TaskController::class, 'rollbackTask'])->name('tasks.rollback_task')->middleware('can:rollback,task');
     Route::post('tasks/{task}/setViewed', [TaskController::class, 'setViewed'])->name('tasks.set_viewed')->middleware('can:viewScreen,task,screen');
     Route::put('tasks/{task}/setPriority', [TaskController::class, 'setPriority'])->name('tasks.priority');
+    Route::get('tasks/getAssignedUsersInGroups', [TaskController::class, 'getAssignedUsersInGroups'])->name('tasks.getAssignedUsersInGroups')->middleware('can:view-users');
```

Comment:
If we consider to implement user retrieval from nested groups, I think it would be a good idea to add a unit test for this new API endpoint

Reply:
Thank you, this has been resolved!